Driver/Driver/Scenes/LoginScreen/ViewController/LoginViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  LoginViewController.swift
+//  Driver
+//
+//  Created by Fatih Doğan on 22.02.2023.
+//
+
+import UIKit.UIViewController
+
+final class LoginViewController: UIViewController {
+    
+    // MARK: Outlets
+    @IBOutlet weak var userNameTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    
+    // MARK: Properties
+    var viewModel: LoginViewModelProtocol!
+    
+    // MARK: LifeCycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        userNameTextField.delegate = self
+        passwordTextField.delegate = self
+        userNameTextField.becomeFirstResponder()
```

Comment:
should be in viewWillAppear or viewDidAppear. It won't work, if Im not wrong.

Reply:
It works in this method but I changed it anyway